content/index.md

```diff
@@ -39,6 +39,7 @@ tools:
   - /susy/
   - /accoutrement/
   - /herman/
+  - /oddcontrast/
```

Comment:
@SondraE What do you think about adding it to the homepage like this? Feel free to remove this if you prefer.